Cargo.lock

```diff
@@ -5028,7 +5039,7 @@ dependencies = [
  "hermit-abi",
  "libc",
  "miniz_oxide",
- "object",
```

Comment:
There were some fixes in 0.23  for some object file formats which are necessary to be here (otherwise the writer will panic). In `std` it will update to 0.24 when https://github.com/rust-lang/backtrace-rs/pull/421 is picked up, and this should be able to move forward at that time as well.